lib/config_builder/model/vm.rb

```diff
@@ -166,8 +172,9 @@ def initialize
   end
 
   def to_proc
+    autostart = attr(:autostart).nil? ? true : attr(:autostart)
```

Comment:
You're absolutely right - we'd looked around for how to set a default but hadn't made the right connection with modifying defaults hash. New commit pushed which takes this approach instead.